src/components/Autocomplete/AutocompleteSearch.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+    <div>
+        <div class="dropdown">
+            <input
+                type="text"
+                class="form-control w-full px-5 custom-select bg-white rounded-md py-4 font-semibold text-lg"
+                v-model="inputValue"
+                @input="searchEvent"
+            />
+            <div v-show="inputValue && apiLoaded" class="dropdown-list">
+                <div v-if="itemList.length === 0" class="dropdown-item">
+                    Inventory not available!!
+                </div>
+                <div
+                    @click="selectItem(item)"
+                    v-for="item in itemList"
+                    :key="item.id"
+                    class="dropdown-item"
+                    v-else
+                >
+                    {{
+                        formatInput(item)
+                    }}
+                </div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+import queryParam from "../../utilities/queryParam";
+import { get, post } from "../../utilities/api";
+
+export default {
+    name: "AutoComplete",
+    props: {
+        apiUrl: "",
+        currentValue: {
+            type: String,
+            default: ""
+        }
+    },
+    data() {
+        return {
+            selectedItem: {},
+            inputValue: "",
+            itemList: [],
+            apiLoaded: false
+        };
+    },
+
+    mounted(){
+        this.inputValue = this.currentValue;
+    },
+
+    methods: {
+        selectItem(data) {
+            this.inputValue = this.formatInput(data);
+            this.apiLoaded = false;
+            this.$emit("childToParent", data);
+        },
+        searchEvent() {
+            this.getproduct();
+        },
+
+        async getproduct() {
+            let query = {};
+            if (this.apiUrl === "/api/inventory") {
```

Comment:
Are you reusing this AutoComplete component?

Reply:
@darmhoo kindly treat